controllers/profileController.js

```diff
@@ -0,0 +1,77 @@
+const User = require("../models/user.js")
+
+const controller = {
```

Comment:
I see you restructured the controller here.  
I personally like that style of creating an object better than what I have now but did it because it would be more readable in a single file especially since we're working together.  
If we do want to change this to be:
```js
const controller = {
  profile: (req, res) => {}
}
```
instead of:
```js
const controller  = {}
controller.profile = (req, res) => {}
```

I prefer the first way just like how you changed it, if Deanna agrees we might also want to change that in the handlebarsHelper-file as well to keep the consistency.

Reply:
Ive included some comments to justify how ive split up the module, i also think they should not be there. So i gues its up to Deanna to decide which of both object declarations is more ledgible :)

Reply:
I think it's a 2-1 already haha, but I also prefer the first one.
It's better in readability, plus I don't like having to consistently declare a bunch of variables first before being able to put stuff inside.